js/app/controllers/calcontroller.js

```diff
@@ -38,6 +38,18 @@ app.controller('CalController', ['$scope', 'Calendar', 'CalendarService', 'VEven
 		$scope.eventModal = null;
 		var switcher = [];
 
+		$scope.$on('ready', () => {
+			var autoOpenNewEvent = $location.search().hasOwnProperty('newevent');
+			if (autoOpenNewEvent) {
+				$scope.fcConfig.select(
+					moment(),
+					moment(),
+					{}, // TODO(leon): Do we need this argument?
```

Comment:
Yes, It's kind of important to determine the position of the element. 

Instead of calling `fcConfig.select` directly, I would emit a `click()` event on todays fullcalendar item.